src/pretix/base/pdf.py

```diff
@@ -546,6 +546,14 @@ def get_variables(event):
     v = copy.copy(DEFAULT_VARIABLES)
 
     scheme = PERSON_NAME_SCHEMES[event.settings.name_scheme]
+
+    concatenation_for_salutation = scheme.get("concatenation_for_salutation", scheme["concatenation"])
+    v['attendee_name_for_salutation'] = {
+        'label': _("Attendee name for salutation"),
+        'editor_sample': _("Mr Doe"),
+        'evaluate': lambda op, order, ev: concatenation_for_salutation(op.attendee_name_parts)
```

Comment:
Is `op.attendee_name_parts` always a dict or could it be None? If so, this line should read

```suggestion
        'evaluate': lambda op, order, ev: concatenation_for_salutation(op.attendee_name_parts or {})
```